src/index.ts

```diff
@@ -9,40 +9,56 @@ export interface Paginated<T> {
   results: T[],
 }
 
+export interface Table {
+  name: string;
+  edge: boolean;
+  id: number;
```

Comment:
Isn't the table ID a string with a slash in it? Or is this a Django ID?

Reply:
The `id` here is the id of the Django `Table` model, which is just an integer. ID's that are strings of the form `doc/key` are just for documents in arango (table rows).